tests/3_endpoints/test_3_0_ldp.py

```diff
@@ -1859,6 +1873,87 @@ def test_delete_no_tstone(self):
+        })
+        assert resp2.status_code == 200
+        self._assert_pref_applied(resp2, include=[Ldpr.RETURN_SRV_MGD_RES_URI])
+
+        resp3 = self.client.get('/ldp/test_contradicting_prefs01', headers={
+            'prefer': 'return=representation; include=\"{0} {1}\"; omit=\"{0} {2}\"'.format(Ldpr.EMBED_CHILD_RES_URI,
+                                                                                            Ldpr.RETURN_SRV_MGD_RES_URI,
+                                                                                            Ldpr.RETURN_INBOUND_REF_URI)
+        })
+        assert resp3.status_code == 200
+        self._assert_pref_applied(resp3, include=[str(Ldpr.RETURN_SRV_MGD_RES_URI)], omit=[str(Ldpr.RETURN_INBOUND_REF_URI)])
+
+    def test_multiple_preferences(self):
+        """
+        Test multiple include and/or omit preferences.
+        """
+        self.client.put('/ldp/test_multiple_preferences01', content_type='text/turtle')
+
+        resp1 = self.client.get('/ldp/test_multiple_preferences01', headers={
+            'prefer': 'return=representation; include=\"{0} {1}\"; omit=\"{2}\"'.format(Ldpr.RETURN_CHILD_RES_URI,
+                                                                                        Ldpr.EMBED_CHILD_RES_URI,
+                                                                                        Ldpr.RETURN_SRV_MGD_RES_URI)
+        })
+        assert resp1.status_code == 200
+        self._assert_pref_applied(resp1, include=[Ldpr.RETURN_CHILD_RES_URI, Ldpr.EMBED_CHILD_RES_URI],
+                                  omit=[Ldpr.RETURN_SRV_MGD_RES_URI])
+
+    def _assert_pref_applied(self, response, value='representation', include=None, omit=None):
+        """
+        Utility to test a response for a Preference-Applied header with the include or omit lists.
+        :param response:: The client response.
+        :param value:: The return=<value>.
+        :param include:: The list of include URIs.
+        :param omit:: The list of omit URIs.
+        :return:
+        """
+        if include is None and omit is None:
+            assert 'Preference-Applied' not in response.headers
+        else:
+            if include is None:
+                include = []
+            if omit is None:
+                omit = []
+            assert 'Preference-Applied' in response.headers
+            headers = toolbox.parse_rfc7240(response.headers['Preference-Applied'])
+            assert headers['return']['value'] == value
+            if value == 'representation':
+                self._assert_pref_header_exists(include, headers['return'], 'include')
+                self._assert_pref_header_exists(omit, headers['return'], 'omit')
+
+    def _assert_pref_header_exists(self, expected, returned, type='include'):
+        """
+        Utility function to compare a list of expected preferences to an include or omit string.
+        :param self:
```

Comment:
Add a blank line between description and parameter listing.
No need to include `self` in parameters doc.